### PR TITLE
use synthetic dataset for test

### PR DESCRIPTION
**Pull request by OleMussmann**
- Create synthetic dataset
- Create new inference output file
- Adjust test suite

### CODE REVIEW

File: tests/integration_test.py [hunk 1]
```diff
@@ -64,5 +64,5 @@ def test_powerchainger_smoke():
 
     df = pd.DataFrame(data, index=ds_parser.index, columns=["inference"])
 
-    expected = np.load(expected_file)
+    expected = np.load(expected_file, allow_pickle=True)
```
Comment (sjvrijn) [inline tests/integration_test.py:67]: Out of curiosity: why include `allow_pickle=True` here?

Comment (OleMussmann) [reply tests/integration_test.py:67]: Huh. During development I needed it so `numpy` does not throw an error. I wanted to look up which one exactly, but now I can't reproduce it anymore. Weird. I'll remove it.

Comment (OleMussmann) [reply tests/integration_test.py]: Some CI tests fail because of errors in the synthetic data. They did pass locally. I'll fix them and update the PR.

Comment (OleMussmann) [reply tests/integration_test.py]: Nice. Now only the SonarCloud CI fails, but that's covered through https://github.com/ESI-FAR/TeSoPs-ELECTRIcity_NILM/pull/44 .

**Review by sjvrijn — APPROVED** [review]
LGTM, thanks!

**Pull request merged**